Doc/library/doctest.rst

```diff
@@ -569,35 +569,20 @@ doctest decides whether actual output matches an example's expected output:
    exception raised is ``ValueError: 3*14``, but will fail, e.g., if
    :exc:`TypeError` is raised.
 
-   It will also ignore the module name used in Python 3 doctest reports. Hence
-   both of these variations will work with the flag specified, regardless of
-   whether the test is run under Python 2.7 or Python 3.2 (or later versions)::
+   It will also ignore the module name used in doctest reports;
+   hence, both of these variations will work with the flag specified::
 
       >>> raise CustomError('message')
       Traceback (most recent call last):
-      CustomError: message
+      my_module.CustomError: message
```

Comment:
Please keep an example without module name.

The module name is not output if it is `"builtins"` or `"__main__"`. Some extension types can have `__module__` incorrectly set to "builtins", because it is the default, it is not rare case.

Reply:
Thanks; I included examples both with, without and with a different module name, using the plausible real-world examples you list above.